src/Connect.js

```diff
@@ -111,16 +103,28 @@ class Uport {
     return this.txObjectHandler(txobj, uriHandler)
   }
 
+  addAppParameters (txObject, callbackUrl) {
```

Comment:
@pelle yeah whatever you feel best describes it, this was a minor point of confusion for me at first glance, so just some alternatives to consider, of course params is not descript either. I think the functional programmer in me, if I could say so :), saw paramsToUri(addAppParameters(txObject) -> params) -> Uri so paramsToUri(txObjecToParams(txObject) -> params) -> Uri made sense, but appParams may be more descript